scripts/asm-installer/install_asm

```diff
@@ -867,6 +877,15 @@ install_asm(){
   sleep 1
   info "...done!"
 
+  if [[ "$DISABLE_CANONICAL_SERVICE" != 1 ]]; then
+    info "Installing ASM CanonicalService controller..."
```

Comment:
It may be worth calling out here that it's going to the asm_system namespace. Someone might delete it if they don't recognize it otherwise.

Reply:
Will add.